internal/cmd/stats.go

```diff
@@ -41,8 +44,10 @@ func stats(provider string, details bool) error {
 
 	fmt.Printf("Provider: %s\n", provider)
 	statsBytes, _ := json.MarshalIndent(schemaStats, "", "  ")
-	statsString := string(statsBytes)
-	fmt.Printf(statsString)
+	_, err = os.Stdout.Write(statsBytes)
+	if err != nil {
```

Comment:
Curious does this ever err.

Reply:
You can close stdout, but it practice this does not.